### PR TITLE
Add docs to bump v1.5.0

### PR DESCRIPTION
**Pull request by SvenDowideit**
`git cherry -v bump master | grep "^+ "` is our friend

I have made the assumption that we are happy to have the new maintainer process docs in this release, if not, feel free to modify :)

```
[sven@t440s docker]$ git cherry -v add-docs-to-bump-v1.5.0 upstream/master | grep "^+ " | wc
     97     842    8453
```

much `git show` and `git cherry-pick` later:

```
[sven@t440s docker]$ git cherry -v add-docs-to-bump-v1.5.0 upstream/master | grep "^+ " | wc
     61     506    5178
[sven@t440s docker]$
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -151,7 +151,7 @@ ENV DOCKER_BUILDTAGS apparmor selinux btrfs_noversion
 COPY vendor /go/src/github.com/docker/docker/vendor
 # (copy vendor/ because go-md2man needs golang.org/x/net)
 RUN set -x \
-	&& git clone -b v1 https://github.com/cpuguy83/go-md2man.git /go/src/github.com/cpuguy83/go-md2man \
+	&& git clone -b v1.0.1 https://github.com/cpuguy83/go-md2man.git /go/src/github.com/cpuguy83/go-md2man \
```
Comment (SvenDowideit) [inline Dockerfile:154]: I think you'll need this to build the man pages for the packages - @cpuguy83 can you please confirm?

Comment (cpuguy83) [inline Dockerfile:154]: It does fix a rendering issue

**Comment by icecrime** [comment]
Awesome @SvenDowideit thank you! Don't worry if you see this PR closed without merging: I might do it "out of GitHub" so I can `rebase -i` to keep my bump commit on top.

**Comment by fredlf** [comment]
LGTM, natch.

**Pull request closed without merging**